packages/app/screens/trip/createTrip.tsx

```diff
@@ -60,131 +61,12 @@ export default function Trips() {
     filteredParks: parksData,
   } = useParks({
     latLng,
-    selectedSearch,
   });
 
-  console.log('filtered parks', parksData, parksError);
   const { data, filteredTrails, error, isLoading } = useTrails({
     latLng,
-    selectedSearch,
+    selectedSearch: osm.name,
   });
-  // useEffect(() => {
-  //   setTrailsData(trailsObject);
-  // }, [trailsObject]);
-
-  // useEffect(() => {
-  // setParksData(parksObject);
-  // }, [parksObject]);
-
-  const { data: photonDetails } = useGetPhotonDetails({
-    properties: {
-      osm_id: searchResult.properties?.osm_id,
-      osm_type: searchResult.properties?.osm_type,
-    },
-  });
-
-  const steps = [
```

Comment:
this is for native side where it appears as a multi step form rather than one large scrollable page